src/jaxsot/core/block_coordinate_descent.py

```diff
@@ -2,12 +2,8 @@
 import jax.numpy as jnp
 import jaxopt
 
-def QP_obj_xk(params,W_x,K,bx,shapeA0):
-    obj = 0.5*jnp.dot(params, jnp.dot((W_x+K),params)) + jnp.dot(bx,params)
-    return obj/shapeA0
-
-def QP_obj_ak(params,W_a,T_a,b,shapeA0):
-    obj = 0.5*jnp.dot(params, jnp.dot((W_a+T_a),params)) + jnp.dot(b,params)
+def QP_objective(params,W,b,shapeA0):
```

Comment:
Can you add a comment with Args and Returns?

Reply:
Thank you. I will add the comment.

Reply:
better to clarify the equation?
"""objective function, Q = 1/2 params^T W x + b^T params, for Quadratic Programming (QP)"""

Reply:
Exactly. I modified.